### PR TITLE
Issue #223: added oc-env step for minishift

### PR DESCRIPTION
**Pull request by rhoads-zach**
@cmoulliard mind taking a look?

### CODE REVIEW

File: docs/topics/minishift-install-prereq.adoc [hunk 1]
```diff
@@ -13,6 +13,19 @@ $ minishift version
 Minishift version: 1.0.0
 ----
 
+. Determine the command to install the correct version of the `oc` binary and run the command.
```
Comment (Ladicek) [inline docs/topics/minishift-install-prereq.adoc]: This doesn't really _install_ `oc`, it just adds correct path to $PATH. I'd suggest rewording to something like "Use `minishift oc-env` to add the correct `oc` binary to your path."

(The following item already uses the expression "your path", so I figured it should be enough. If a link to explanation is needed, this Wikipedia page is good: https://en.wikipedia.org/wiki/PATH_(variable) :-) )

Comment (Ladicek) [inline docs/topics/minishift-install-prereq.adoc]: Actually with this paragraph added, the WARNING below (line 40) seems no longer necessary, or it could be shortened and moved to this section?

**Comment by rhoads-zach** [comment]
@Ladicek updated per feedback. Let me know what you think, otherwise, i'll merge later today

**Comment by rhoads-zach** [comment]
Thanks @Ladicek, merging.

**Comment by rhoads-zach** [comment]
resolves #223
